bin/python/ripple/util/test_Sign.py

```diff
@@ -59,13 +52,17 @@ def test_make_manifest(self):
             '\xef*\x97\x16n<\xa6\xf2\xe4\xfb\xfc\xcd\x80P[\xf1s\x06verify')
 
     def test_sign_manifest(self):
+        ssk, spk = Sign.make_ecdsa_keypair(self.urandom)
```

Comment:
For future reference, the `-v` flag will output a more verbose running of the tests. So `python -m unittest discover -v` will show you what tests are being run. To run an individual test: `python -m unittest ripple.util.test_Sign`